src/ralph/api/backends/statements/mongo.py

```diff
@@ -0,0 +1,19 @@
+"""MongoDB xAPI statements route backend."""
+
+from ralph.backends.database.mongo import MongoDatabase
+
+from .base import BaseStatementsBackend, StatementParameters, StatementQueryResult
+
+
+class MongoStatementsBackend(BaseStatementsBackend, MongoDatabase):
+    """MongoDB xAPI statements route backend."""
+
+    def query_statements(self, params: StatementParameters) -> StatementQueryResult:
+        """Returns the results of a statements query using xAPI parameters."""
+
+        raise Exception("TODO")
```

Comment:
```suggestion
        raise NotImplemented
```

Reply:
Good Idea) Thanks) these methods will be implemented in this pull request)